csharp/extractor/Semmle.Extraction.CSharp/SymbolExtensions.cs

```diff
@@ -117,7 +117,7 @@ bool IdDependsOnImpl(ITypeSymbol type)
                         case TypeKind.Delegate:
                         case TypeKind.Error:
                             var named = (INamedTypeSymbol)type;
-                            if (named.IsTupleType)
+                            if (named.IsTupleType && named.TupleUnderlyingType is object)
```

Comment:
`Tuple<T1,T2,T3,T4,T5,T6,T7,TRest>` and similar are `IsTupleType  == true`, but `TupleUnderlyingType  == null`